README.md

```diff
@@ -30,6 +30,17 @@ const ContentBasedRecommender = require('content-based-recommender')
 
 ## What's New
 
+#### 1.5.0
+
+* Added `trainBidirectional(collectionA, collectionB)` to allow recommendations between
+two different datasets
+* Additional metadata can now be added to documents and will be passed through the process
+````
+const documents = [
+  { id: '1000001', content: 'Why studying javascript is fun?', meta:{key:'Hello'}, test: 'Another Param' },
+]
+````
+
```

Comment:
Thanks adding to the changelog.

Would you mind also updating the bottom part of the readme about how to use the newly implemented method?